Resources/Prototypes/Wires/layouts.yml

```diff
@@ -75,3 +75,14 @@
   wires:
   - !type:PowerWireAction
   - !type:CryoPodEjectLockWireAction
+
+- type: wireLayout
+  id: Defusable
+  dummyWires: 2
+  wires:
+    - !type:ActivateBombWireAction
+#    - !type:BoltBombWireAction
+#    - !type:DelayBombWireAction
+#    - !type:ProceedBombWireAction
+    - !type:BoomWireAction
```

Comment:
surely you can come up with a better name than this

Reply:
what's wrong with boom wire

Reply:
boom is 4 letters so some qwrky shorthand not needed

Reply:
> the rest have descriptive names then there's boomwire just do like, `DetonateBombWireAction`

how is "boomwire" not descriptive enough